app/views/home/first_login_help.html.erb

```diff
@@ -1,49 +1,233 @@
+  set_preferences: (link_to ts('Set My Preferences'),     
+                            user_preferences_path(current_user)),
+  edit_profile: (link_to ts('Edit My Profile'), edit_user_path(current_user)),
+  preferences_faq: (link_to ts('Preferences FAQ'),
+                            archive_faqs_path + '/preferences') 
+  ).html_safe %></p>
+  <p><%= ts('For more information on how to customize the skins and interface of 
+  the Archive, refer to the %{skins_faq} and %{tutorial_list}.',
+  skins_faq: (link_to ts('Skins and Archive Interface FAQ'), 
+                      archive_faqs_path + '/skins-and-archive-interface'),
+  tutorial_list: (link_to ts('List of Tutorials'), 
+                          archive_faqs_path + '/tutorials')
+  ).html_safe %></p>
+
+  <h3 class="heading" id="additional">
+    <%= ts('Additional Browsing Information') %>
+  </h3>
+  <h4 class="heading" id="history">
+    <%= ts('History and Mark for Later') %></h4>
+  <p><%= ts('You can access and manage your history by going to %{dashboard} and 
+  selecting the "%{history}" link in the side menu in the default browser skin 
+  or on top of the page in mobile devices. You can clear your history, or delete 
+  individual entries if you don\'t want these entries to show up in your 
+  history. You can also add works to the "Marked for Later" list in your 
+  history. For more information on History, please visit the %{history_faq}.',
+  dashboard: (link_to ts('your Dashboard'), user_path(current_user)),
+  history: (link_to ts('History'), user_readings_path(current_user)),
+  history_faq: (link_to ts('History and Mark for Later FAQ'), 
+                        archive_faqs_path + '/History-and-mark-for-later')
+  ).html_safe %></p>
+
+  <h4 class="heading" id="subscriptions">
+    <%= ts('Subscriptions') %>
+  </h4>
+  <p><%= ts('You can subscribe to a work, collection, series of works, or user 
+  by selecting the "Subscribe" link at the top or bottom of a work, or at the 
+  top or bottom of a collection or series page, or the user\'s profile page. You 
+  can check your subscriptions from %{dashboard} by selecting the 
+  "Subscriptions" link. For more information on Subscriptions, please visit the 
+  %{subscription_faq}.',
+  dashboard: (link_to ts('your Dashboard'), user_path(current_user)),
+  subscription_faq: (link_to ts('Subscriptions and Feeds FAQ'), 
+                             archive_faqs_path + '/subscriptions-and-feeds')
+  ).html_safe %></p>
+
+  <h3 class="heading" id="legal_stuff">
+    <%= ts('Terms of Service') %>
+  </h3>
+  <p><%= ts('You can keep up to date with the policies and procedures for the 
+  site listed in the current %{tos}, and find explanations for some of the more 
+  common questions in the %{tos_faq}. If you still have questions about a 
+  specific work, please %{contact_abuse}. If you have general policy questions, 
+  you can either contact Abuse or %{contact_support}.',
```

Comment:
I think it's probably okay given there's already a link just a few words before. If it were a different section, I'd definitely say yes.